scripts/release.js

```diff
@@ -156,6 +156,19 @@ logger.warn(`
 
 shouldContinuePrompt();
 
+// Creates a new React test app, install the component app as dependency and build the test app to ensure no errors or build issues
+process.chdir(path.join(__dirname, '..'));
+spawnOrFail('yalc', ['publish']);
```

Comment:
Make sure to verify the content of yalc publish. You need to run npm install && npm run build before yalc publish otherwise it would be a empty package. We may already do this above but just make sure.

Reply:
Will do. Thanks for pointing this out.